vsintegration/tests/UnitTests/LegacyLanguageService/Tests.LanguageService.Completion.fs

```diff
@@ -783,7 +783,7 @@ for i in 0..a."]
                 "let r = {AAA = 5}"
                 "let b = {r  with }"
             ]
-        AssertCtrlSpaceCompleteContains code "{r " [] ["AAA"]
```

Comment:
Humm. This will enabled completion before the `with` keyword.  To fix this we can do some range calculations in `ServiceParseTreeWalk.fs`  and update the completion context for records or capture the `with` range in the tree and then use it.